### DISCUSSION TITLE
Windows app 414 is better then 428

### DISCUSSION THREAD

**Discussion by Onair-santa** [vpnhood/VpnHood#392] [General]
Autohide is very good
Why do we need extra pop-up icons in the taskbar? Fixed windows. For what?
Strange decision. I left version 414-416 for all clients because there were a lot of complaints

**Comment by trudyhood** [comment]
Hi, thank you for sharing this with me. Here are some reasons why the new Windows app is behaving this way:

* The app's presence in the Windows 10 or upper taskbar shows the current state of the application and also serves as a convenient shortcut for quick access, which is not much different from the system tray. Even though the system tray icon may be smaller, it still occupies space on the taskbar if it doesn't hide in the drawer.

* Effective Management by Windows: Windows can effectively manage the app's presence in the taskbar. We are trying to make the app more standard with the new Windows 10 and 11 UI and more tablet-friendly.

You can find the same behavior from new apps like Telegram and WhatsApp! However, we understand that the app icon's behavior may be inconvenient for users who are not accustomed to it. To address this concern, users have the option to close the VpnHood window, which will keep it in the notification area while removing the icon from the taskbar. We acknowledge that users may not be aware of this closing behavior, so we will be adding an option in the settings to allow users to minimize the app into the system tray intentionally. This way, users can choose the behavior that suits their preferences.

So the simple solution is "Just close the window, and it will go into the taskbar till we add an autohide option on minimize." 

If you have any more comments, we are always here to listen.
Good luck
